xdesign/geometry.py

```diff
@@ -67,8 +67,6 @@
 __docformat__ = 'restructuredtext en'
 __all__ = ['Entity',
            'Point',
-        #    'Superellipse',
```

Comment:
Hello, I wonder why you deleted the ellipse here. Sometimes I need to add an ellipse phantom.

Reply:
Hi @huyuhuster, there is [no algorithm implemented](https://github.com/tomography/xdesign/blob/master/src/xdesign/geometry/intersect.py) to compute the area of intersection between a rectangle and a superellipse. Therefore, these entities cannot be used in phantoms for data acquisition simulation. If you want to try to use these entities anyways, they are still there, just removed from `__all__`; you can import then directly.

Reply:
Hi @carterbox , thanks for your reply. Do you plan to implement the algorithm? By the way, when I simulating the acquisition, it will be very slow if the size of the probe is small, such as 1/1000. Do you have any method to speed it? Often I need to cost several hours to simulate one phantom.

Reply:
I have no such plans because I'm working on other things. PRs are welcome though. 😁

Reply:
OK，do you have any suggestion of speed the acquisition simulation?

Reply:
No. I haven't looked at this codebase in more than a year.